pkg/providers/tinkerbell/reconciler/reconciler_test.go

```diff
@@ -44,7 +53,7 @@ func TestReconcilerReconcileSuccess(t *testing.T) {
 	tt.ipValidator.EXPECT().ValidateControlPlaneIP(tt.ctx, logger, tt.buildSpec()).Return(controller.Result{}, nil)
 
 	tt.remoteClientRegistry.EXPECT().GetClient(
-		tt.ctx, client.ObjectKey{Name: "workload-cluster", Namespace: "eksa-system"},
+		tt.ctx, client.ObjectKey{Name: "workload-cluster", Namespace: constants.EksaSystemNamespace},
```

Comment:
Not super necessary but you could assign assign "workload-cluster" to a variable since you use it multiple times within this function.